index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style/style.css">
+    <title>First Capstone</title>
+</head>
+<body>
+
+    <!-- mobile menu -->
+    <div class="menuformobile">
+        <img class="closeicon" onclick="CloseMenu()" src="style/images/Icon.png" alt="close_icon">
+        <div class="top-menu">
+            <ul>
+                <li><a><i class="fa-brands fa-facebook"></i></a></li>
+                <li><a><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a>Logout</a></li>
+                <li><a>Mypage</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/Morocco">Morocco</a></li>
+            </ul>
+        </div>
+        <ul class="mobile-menu">
+            <li><a href="./index.html" class="links" id="homeLink" >Home</a></li>
+            <li><a href="./about.html" class="links" id="aboutLink" >About</a></li>
+            <li><a id="programLink" class="links">Program</a></li>
+            <li><a id="joinLink" class="links">Join</a></li>
+            <li><a id="contactLink" class="links">Contact</a></li>
+        </ul>
+    </div>
```

Comment:
- [OPTIONAL] Kindly make sure to wrap your navigation bar into the `header` tag. :wink: 
- [OPTIONAL] Kindly make sure to wrap every time the navigation bar into the nav tag in order to structure your HTML semantically. wink